pyproject.toml

```diff
@@ -288,3 +288,383 @@ known_zope = "zope"
+    'twisted.test.test_htb',
+    'twisted.test.test_ident',
+    'twisted.test.test_internet',
+    'twisted.test.test_iosim',
+    'twisted.test.test_iutils',
+    'twisted.test.test_lockfile',
+    'twisted.test.test_log',
+    'twisted.test.test_logfile',
+    'twisted.test.test_loopback',
+    'twisted.test.test_main',
+    'twisted.test.test_memcache',
+    'twisted.test.test_modules',
+    'twisted.test.test_monkey',
+    'twisted.test.test_paths',
+    'twisted.test.test_pcp',
+    'twisted.test.test_persisted',
+    'twisted.test.test_policies',
+    'twisted.test.test_postfix',
+    'twisted.test.test_process',
+    'twisted.test.test_protocols',
+    'twisted.test.test_randbytes',
+    'twisted.test.test_rebuild',
+    'twisted.test.test_reflect',
+    'twisted.test.test_roots',
+    'twisted.test.test_shortcut',
+    'twisted.test.test_sip',
+    'twisted.test.test_sob',
+    'twisted.test.test_socks',
+    'twisted.test.test_ssl',
+    'twisted.test.test_sslverify',
+    'twisted.test.test_stateful',
+    'twisted.test.test_stdio',
+    'twisted.test.test_strerror',
+    'twisted.test.test_strports',
+    'twisted.test.test_task',
+    'twisted.test.test_tcp',
+    'twisted.test.test_tcp_internals',
+    'twisted.test.test_text',
+    'twisted.test.test_threadable',
+    'twisted.test.test_threadpool',
+    'twisted.test.test_threads',
+    'twisted.test.test_tpfile',
+    'twisted.test.test_twistd',
+    'twisted.test.test_twisted',
+    'twisted.test.test_udp',
+    'twisted.test.test_unix',
+    'twisted.test.test_usage',
+    'twisted.test.testutils',
+    'twisted.trial.*',
+    'twisted.web.*',
+    'twisted.words.*',
+]
+
+[[tool.mypy.overrides]]
+allow_untyped_defs = true
+module = [
+    'twisted.application.test.test_service',
+    'twisted.python._pydoctor',
+    'twisted.python._release',
+    'twisted.python._shellcomp',
+    'twisted.python._textattributes',
+    'twisted.python.compat',
+    'twisted.python.components',
+    'twisted.python.context',
+    'twisted.python.deprecate',
+    'twisted.python.fakepwd',
+    'twisted.python.htmlizer',
+    'twisted.python.lockfile',
+    'twisted.python.log',
+    'twisted.python.monkey',
+    'twisted.python.procutils',
+    'twisted.python.randbytes',
+    'twisted.python.release',
+    'twisted.python.roots',
+    'twisted.python.shortcut',
+    'twisted.python.syslog',
+    'twisted.python.test.test_dist3',
+    'twisted.python.test.test_htmlizer',
+    'twisted.python.test.test_runtime',
+    'twisted.python.test.test_setup',
+    'twisted.python.test.test_textattributes',
+    'twisted.python.test.test_tzhelper',
+    'twisted.python.test.test_versions',
+    'twisted.python.test.test_win32',
+    'twisted.python.test.test_zippath',
+    'twisted.python.threadpool',
+    'twisted.python.zippath',
+    'twisted.runner.inetdtap',
+    'twisted.runner.procmontap',
+    'twisted.runner.inetdconf',
+    'twisted.runner.test.test_procmontap',
+]
+
```

Comment:
```suggestion
```

pretty sure this config block is redundant

Reply:
Feel free to delete this.
As long as mypy check is green, this should be ok.

Thanks